### PR TITLE
SUS Bathroom, Part 1

### PR DESCRIPTION
**Pull request by xanderrootslayer**
#### Summary

```SUMMARY: FEATURES "13 new items, complete with crafting and disassembling, are being added with future plans to designate spawn locations. Improvised Shaving Kit has slightly more plausible contents."```

#### Purpose of change

"The Specific Use Storage project started by I-Am-Erk aims to make residential houses more realistic by adding plausible clutter times to houses. The project started with adding plates, dishes and utensils to kitchens, but it needs more content all the time. Bathrooms are a natural next step due to the sheer amount of plastic rubbish they accumulate. Also, this adds a few mechanical items which are useless without a power grid, but can still be scrapped for parts.

#### Describe the solution

Adding item spawns will take quite a bit of testing, so we need the items to exist in the game's files first. Never make a pull request bigger than your own head.

#### Describe alternatives you've considered

Attempting to release all the intended changes at once would delay any features or content considerably. Also, the Soap Dish is completely pointless but it's funny. 

#### Testing

The usual, create a Black Belt, spawn in the items and make sure all the interactions are possible.

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
*started by @I-am-Erk

File: data/json/items/generic/bathroom_house.json [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        "id": "hairbrush",
+        "name": "hairbrush",
+        "name_plural": "hairbrushes",
+        "type": "GENERIC",
+        "description": "An instrument of hair torture.",
+        "copy_from": "base_toiletries",
+        "snippet_category": [
+            { "id:": "hbrush1", "text": "There are round safety tips on the bristles."},
+            { "id:": "hbrush2", "text": "The handle has a faux-wood design."},
+            { "id:": "hbrush3", "text": "The shiny crome design makes it look modern."},
```
Comment (FuelType-Memes) [inline data/json/items/generic/bathroom_house.json]: "crome" -> "chrome"?

File: data/json/recipes/recipe_deconstruction.json [hunk 2]
```diff
@@ -3057,5 +3057,49 @@
...
+    "components": [ [ [ "plastic_chunk", 1 ] ], [ [ "razor_blade", 1 ] ] ]
+  },
+  {
+    "result": "hair_dryer",
+    "type": "uncraft",
+    "skill_used": "mechanics",
+    "difficulty": 1,
+    "time": "1 m",
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "components": [ [ [ "plastic_chunk", 2 ] ], [ [ "motor_micro", 1 ] ], [ [ "cable", 1 ] ], [ [ "wire", 1 ] ] ]
```
Comment (FuelType-Memes) [inline data/json/recipes/recipe_deconstruction.json]: maybe add heating element?

File: data/json/recipes/recipe_deconstruction.json [hunk 3]
```diff
@@ -3057,5 +3057,49 @@
...
+    "//": "just unspool the floss, no need to make a big production out of it.",
+    "components": [ [ [ "plastic_chunk", 1 ] ], [ [ "thread", 25 ] ] ]
+  },
+  {
+    "result": "plunger_toilet",
+    "type": "uncraft",
+    "skill_used": "fabrication",
+    "difficulty": 0,
+    "time": "6 s",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
```
Comment (FuelType-Memes) [inline data/json/recipes/recipe_deconstruction.json]: sharp plunger?

File: data/json/items/generic/bathroom_house.json [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+    {
+        
+        "id": "razor_shaving",
+        "name": "shaving razor",
+        "type": "GENERIC",
+        "description": "A razor blade on a comfortable handle. Much easier to shave with than a loose razor.",
+        "copy_from": "base_toiletries",
+        "price": 800,
+        "price_postapoc": 1200,
+        "//": "the end of the world doesn't mean everyone is okay with having ZZ Top beards. disposable razors are a hot commodity. The razor also loses the Cutting 1 quality. Don't ask how we confirmed this.",
```
Comment (LaVeyanFiend) [inline data/json/items/generic/bathroom_house.json]: > The razor also loses the Cutting 1 quality. Don't ask how we confirmed this.

Imma do it! How'd you confirm that 😛

File: data/json/items/generic/bathroom_house.json [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+        "price_postapoc": 0,
+        "weight": 160,
+        "volume": 2,
+        "material": [ "steel", "plastic" ],
+        "bashing": 3,
+        "cutting": 0,
+        "to_hit": 1
+    }
+    
+]
```
Comment (LaVeyanFiend) [inline data/json/items/generic/bathroom_house.json]:
```suggestion
]

```

File: data/json/items/generic/bathroom_house.json [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+        "//": "the end of the world doesn't mean everyone is okay with having ZZ Top beards. disposable razors are a hot commodity. The razor also loses the Cutting 1 quality. Don't ask how we confirmed this.",
+        "weight": 110,
+        "material": [ "plastic", "steel" ],
+        "cutting": 2
+        
+    },
+    
+    {
+        "id": "toothbrush_plain",
+        "name": "toothbrush",
```
Comment (LaVeyanFiend) [inline data/json/items/generic/bathroom_house.json]: Can you add a recipe for the sharpened toothbrush with this? 🎉

File: data/json/items/generic/bathroom_house.json [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+            { "id:": "hbrush3", "text": "The shiny crome design makes it look modern."},
+            { "id:": "hbrush4", "text": "The cartoon whale on the handle seems friendly enough."}
+        ]
+    },
+    
+    {
+        "id": "curler_hair",
+        "name": "hair curler",
+        "type": "GENERIC",
+        "description": "a soft plastic cylinder you can wrap a lock of your hair around to curl it.",
```
Comment (matskuman5) [inline data/json/items/generic/bathroom_house.json]:
```suggestion
        "description": "A soft plastic cylinder you can wrap a lock of your hair around to curl it.",
```

File: data/json/items/generic/bathroom_house.json [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+        "volume": "25 ml",
+        "container": "box_small"
+    },
+    
+    {
+        "id": "string_floss",
+        "name": "dental floss",
+        "name_plural": "rolls of dental floss",
+        "type": "GENERIC",
+        "description": "Twenty five yards of finely waxed thread wound up inside a plastic container. Perfect for picking bits of smoked meat out of your teeth. Disassemble to use the thread for something else.",
```
Comment (matskuman5) [inline data/json/items/generic/bathroom_house.json]:
```suggestion
        "description": "Twenty-five yards of finely waxed thread wound up inside a plastic container. Perfect for picking bits of smoked meat out of your teeth. Disassemble to use the thread for something else.",
```

File: data/json/items/generic/bathroom_house.json (hunk 6 above)
Comment (I-am-Erk) [reply data/json/items/generic/bathroom_house.json]: I'd say that is beyond the scope of this pr but absolutely should be done

File: data/json/items/generic/bathroom_house.json [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+        "bashing": 5,
+        "cutting": 0,
+        "to_hit": -1
+    },
+    
+    {
+        "id": "plunger_futuristic",
+        "name": "professional plunger",
+        "type": "GENERIC",
+        "description": "This hollow plastic toilet plunger's bell compresses like an accordion. It is efficient at its intended purpouse, and complete rubbish as a weapon.",
```
Comment (I-am-Erk) [inline data/json/items/generic/bathroom_house.json]: > Intended purpouse

You mean purpose

File: data/json/recipes/recipe_deconstruction.json [hunk 10]
```diff
@@ -3057,5 +3057,49 @@
...
+    "components": [ [ [ "plastic_chunk", 2 ] ], [ [ "splinter", 1 ] ] ]
```
Comment (I-am-Erk) [inline data/json/recipes/recipe_deconstruction.json]: I think it's safe to either yield a stick, or 5-10 splinters

File: data/json/items/generic/bathroom_house.json (hunk 6 above)
Comment (xanderrootslayer) [reply data/json/items/generic/bathroom_house.json]: that's part of another thing I wanted to do later actually

File: data/json/recipes/recipe_deconstruction.json (hunk 10 above)
Comment (xanderrootslayer) [reply data/json/recipes/recipe_deconstruction.json]: judging by weight it's one Splinter, but if you say so?

File: data/json/recipes/recipe_deconstruction.json (hunk 2 above)
Comment (xanderrootslayer) [reply data/json/recipes/recipe_deconstruction.json]: I'll replace the Wire with a Heating Element. It's not exactly enough wire to build a shopping basket, no...

File: data/json/items/generic/bathroom_house.json (hunk 4 above)
Comment (xanderrootslayer) [reply data/json/items/generic/bathroom_house.json]: There's a bit of abstraction here where the old school razor blade transmogrifies into a three-bladed safety razor when you attach it. Proprietary replacement razors cost like 25 dollars a package in real life, let us have a little bit of escapism...

File: data/json/recipes/recipe_deconstruction.json (hunk 10 above)
Comment (I-am-Erk) [reply data/json/recipes/recipe_deconstruction.json]: I'd guess the weight of a splinter is more the problem there, but I wouldn't worry too much about that.

**Comment by kevingranade** [comment]
This needs double spaces after periods throughout.

**Comment by I-am-Erk** [comment]
If no one objects I will clean up the remaining problems so this can be merged for the sus expansion project

**Comment by I-am-Erk** [comment]
OK, this is largely ready to merge now. I don't see any reason it needs to be in the feature freeze, it doesn't add any features.

File: data/json/recipes/recipe_deconstruction.json [hunk 11]
```diff
@@ -518,13 +518,15 @@
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "components": [
       [ [ "ai_module", 1 ] ],
+      [ [ "identification_module", 1 ] ],
+      [ [ "pathfinding_module", 1 ] ],
+      [ [ "sensor_module", 1 ] ],
```
Comment (LaVeyanFiend) [inline data/json/recipes/recipe_deconstruction.json]: Why are robot deconstruction recipes in here? 🤔

File: data/json/recipes/recipe_others.json [hunk 12]
```diff
@@ -4439,6 +4439,15 @@
...
     "type": "recipe",
+    "result": "razor_shaving",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "time": "30 s",
+    "reversible": true,
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "razor_blade", 1 ] ], [ [ "plastic_chunk", 1 ] ] ]
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_others.json]:
```suggestion
    "components": [ [ [ "razor_blade", 1 ] ], [ [ "plastic_chunk", 1 ] ] ]
    },
    {
      "type": "recipe",
```

File: data/json/recipes/recipe_others.json [hunk 13]
```diff
@@ -4439,6 +4439,18 @@
...
+    },
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_others.json]:
```suggestion
  },
```

File: data/json/recipes/recipe_others.json [hunk 14]
```diff
@@ -4439,6 +4439,18 @@
...
+    {
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_others.json]:
```suggestion
  {
```

File: data/json/recipes/recipe_others.json [hunk 15]
```diff
@@ -4439,6 +4439,18 @@
...
+      "type": "recipe",
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_others.json]:
```suggestion
    "type": "recipe",
```

File: data/json/items/generic/bathroom_house.json [hunk 16]
```diff
@@ -0,0 +1,233 @@
...
+    "weight": 100,
+    "volume": "50 ml",
+    "material": "plastic",
+    "bashing": 1,
+    "to_hit": -1
+  },
+  {
+    "id": "soap_holder",
+    "name": "soap dish",
+    "name_plural": "soap_dishes",
```
Comment (snipercup) [inline data/json/items/generic/bathroom_house.json]:
```suggestion
    "name": { "str": "soap dish", "str_pl": "soap dishes" },
```
Please replace name and name_plural with this format

**Comment by I-am-Erk** [comment]
There, let's get these into some danged item groups now!

**Pull request merged**